### DISCUSSION TITLE
Play/pause with spacebar and arrows for back/forth not working properly.

### DISCUSSION THREAD

**Discussion by SentientBeingOnAQuest** [FreeTubeApp/FreeTube#4226] [Q&A] [answered]
Play/pause with spacebar and arrows for back/forth only works if I manually adjust and pause a few times first. 
I would like them to work from the start. Is there any way I can do this?

**Comment by kommunarr** [comment, marked as answer]
We are tracking this issue at #4137.

**Reply by SentientBeingOnAQuest** [reply]
Ok so if this issue is known and being solved I'll just mark this as solved for now.
